teams.json

```diff
@@ -38,6 +28,33 @@
         "GoogleCloudPlatform/stackdriver-sandbox"
```

Comment:
This needs to be renamed to GoogleCloudPlatform/cloud-ops-sandbox

Reply:
renamed in teams.json and repos.json